runner.go

```diff
@@ -137,6 +138,16 @@ func RunSplunk() bool {
 	return ctx.Err() == nil
 }
 
+func TailFile() bool {
+	args := []string{"-F", os.ExpandEnv(SplunkdLogPath)}
+	tail = exec.CommandContext(ctx, "/usr/bin/tail", args...)
+	tail.Stdout = os.Stderr
+	tail.Stderr = os.Stderr
+	tail.Start()
+	tail.Wait()
```

Comment:
If we're not going to do anything with the errors returned from these functions we should explicitly drop them:
```suggestion
	_ = tail.Start()
	_ = tail.Wait()
```
I wonder if something can be done with returning them to the caller instead of the `ctx.Err() == nil` boolean, but that will have to be explored in the future.